packages/webapp/pages/posts/[id].tsx

```diff
@@ -50,30 +45,27 @@ export interface Props {
   postData?: PostData;
 }
 
+const CONTENT_MAP: Record<PostType, typeof PostContent> = {
+  article: PostContent,
+  share: SquadPostContent,
+};
+
 interface PostParams extends ParsedUrlQuery {
   id: string;
 }
 
 const PostPage = ({ id, postData }: Props): ReactElement => {
   const [position, setPosition] =
     useState<CSSProperties['position']>('relative');
-  const { tokenRefreshed } = useContext(AuthContext);
   const router = useRouter();
   const { isFallback } = router;
 
-  if (!isFallback && !id) {
-    return <Custom404 />;
-  }
+  if (!isFallback && !id) return <Custom404 />;
```

Comment:
Just be aware this will clash with my changes, but shouldn't be breaking to much.

Reply:
I think we should that PR to this branch as there are a lot of changes involved here so it will be easier to fix merge issues. But let's see what fits.

Reply:
Yeah let's do this one first, and i'll adjust from merged :)